plugin/src/lib.rs

```diff
@@ -596,7 +596,7 @@ impl<'a, 'cx> Folder for MutatorPlugin<'a, 'cx> {
 
             },
             e => {
-                P(fold::noop_fold_expr(e, self))
+                P(self.fold_expr(P(e)))
```

Comment:
This one should stay as it is, lest we get an infinite loop.

Reply:
Ah, of course!